examples/servicenow_examples.md

```diff
@@ -11,49 +11,50 @@ added later.
 ```python
 from ticketutil.servicenow import ServiceNowTicket
 
-table = 'x_redha_pnt_devops_table'
-server = 'https://envqa.service-now.com'
-user = 'test_servicenow'
-pwd = 'passwordString'
+table = <table_name>
+server = <servicenow_url>
+user = <user>
+pwd = <password>
 auth = (user,pwd)
 
-ticket = ServiceNowTicket(server, table, auth)
+t = ServiceNowTicket(server, table, auth)
 description = ('This is just a simple test for ticketutil library. Calling '
                 'ServiceNowTicket class you are able to create tickets in '
                 'the ServiceNow automatically. You need however to provide '
                 'username and password (at the moment only basic auth is '
                 'supported. Module created by pzubaty@redhat.com')
 
-ticket.create(short_description='TEST adding SNow API into ticketutil',
-                description=description,
-                u_category='Communication',
-                u_item='ServiceNow')
-ticket.close_requests_session()
+t.create(short_description='TEST adding SNow API into ticketutil',
+         description=description,
+         category='Communication',
+         item='ServiceNow')
+t.close_requests_session()
 ```
 
 Provided `user` and `pwd` are replaced with valid values, similar response
 should be seen:
 
 ```
-INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): redhatqa.service-now.com
+INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): <servicenow_url>
 INFO:root:Successfully authenticated to ServiceNow
-INFO:root:Created issue <number> - <service_now_url>
+INFO:root:Created issue <number> - <ticket_url>
 ```
 
 #### Create new comment, edit ticket and change status
 ```python
 from ticketutil.servicenow import ServiceNowTicket
 
-table = 'x_redha_pnt_devops_table'
-server = 'https://envqa.service-now.com'
-user = 'test_servicenow'
-pwd = 'passwordString'
+table = <table_name>
+server = <servicenow_url>
+user = <user>
+pwd = <password>
 auth = (user,pwd)
 
-ticket = ServiceNowTicket(server, table, auth, ticket_id)
-ticket.add_comment('Lorem ipsum dolor TEST')
-fields = {'assigned_to':'pzubaty', 'priority':'3'}
-ticket.edit(**fields)
-ticket.change_status('Pending')
-ticket.close_requests_session()
+t = ServiceNowTicket(server, table, auth, ticket_id)
+t.add_comment('Lorem ipsum dolor TEST')
+ticket.edit(assigned_to = 'pzubaty', priority = '3',
+            email_from = 'pzubaty@redhat.com',
+            hostname_affected = '127.0.0.1')
```

Comment:
There should be no spaces around `=` in keyword arguments https://www.python.org/dev/peps/pep-0008/#other-recommendations